src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,163 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node head;
+    private Node tail;
```

Comment:
add parametrization everywhere, where you see this warning
![image](https://github.com/mate-academy/jv-linked-list/assets/49767049/9124d07b-a56f-4a89-afde-8057759ed1d7)